runtime/doc/eval.txt

```diff
@@ -10573,5 +10573,124 @@ This is not allowed when the textlock is active:
 	- closing a window or quitting Vim
 	- etc.
 
+==============================================================================
+13. Command-line expressions coloring			*expr-coloring*
+
+Expressions entered by user in |i_CTRL-R_=|, |c_CTRL-\_e|, |quote=| are 
+colored by built-in expressions parser.  It uses highlight groups described in 
+the table below, which may be overriden by user colorschemes, all linked to 
+some other highlighting group.
+							*hl-NVimInvalid*
+In addition to highlighting groups prefixed with NVim described below there 
```

Comment:
I don't like to nitpick about this, but here we are introducing yet another variant of "Nvim" spelling. Same with the highlight group names.